### PR TITLE
fix: add address prefix

### PR DESCRIPTION
**Pull request by ryanseee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,18 +218,34 @@ open-attestation deploy token-registry "My Sample Token" MST --network ropsten
...
+```
+
 #### Issue document to token registry
 
-Issue a hash to a token registry deployed on the blockchain
+`Issue` a hash to a token registry deployed on the blockchain. The `tokenId` option would be used to indicate the document hash, and the `to` option to indicate the title escrow address the document is mapped to.
 
 ```bash
-open-attestation token-registry issue --address <TOKEN_REGISTRY_ADDRESS> --tokenId <TOKEN_ID> --to <TO> [options]
+open-attestation token-registry issue --network <NETWORK> --address <TOKEN_REGISTRY_ADDRESS> --tokenId <TOKEN_ID> --to <TO> [options]
```
Comment (simboonlong) [inline README.md:242]: if im not wrong need `0x` prefix for TOKEN_ID <- https://www.openattestation.com/docs/transferable-record/issuing-transferable-record#issuing-transferable-records

File: src/utils.ts [hunk 2]
```diff
@@ -3,6 +3,8 @@ import chalk from "chalk";
 export const getEtherscanAddress = ({ network }: { network: string }): string =>
   `https://${network === "mainnet" ? "" : `${network}.`}etherscan.io`;
 
+export const validateAddressPrefix = (address: string): string => (address.startsWith("0x") ? address : `0x${address}`);
+
```
Comment (simboonlong) [inline src/utils.ts:7]: it's not really called validating, more of adding `0x` prefix. https://ethereum.stackexchange.com/questions/503/what-is-the-0x-i-see-around-ethereum-for-example-i-see-addresses-that-start-wit

**Comment by simboonlong** [comment]
do remember to change commit message from `docs` to `fix`

File: src/utils.ts (hunk 2 above)
Comment (ryanseee) [reply src/utils.ts]: ok, changed it to addAddressPrefix()

File: src/utils.ts [hunk 3]
```diff
@@ -3,6 +3,8 @@ import chalk from "chalk";
 export const getEtherscanAddress = ({ network }: { network: string }): string =>
   `https://${network === "mainnet" ? "" : `${network}.`}etherscan.io`;
 
+export const addAddressPrefix = (address: string): string => (address.startsWith("0x") ? address : `0x${address}`);
+
```
Comment (simboonlong) [inline src/utils.ts:7]: could add some small tests, otherwise lgtm

**Review by simboonlong — APPROVED** [review]

**Pull request merged**